funnel/util.py

```diff
@@ -17,3 +18,25 @@ def geonameid_from_location(text):
         geonameids = [field['geoname']['geonameid'] for field in response['result'] if 'geoname' in field]
         return set(geonameids)
     return None
+
+
+def format_twitter_handle(handle):
+    """
+    formats a user given twitter handle
+    Eg:  -> shreyas_satish, @shreyas_satish -> shreyas_satish
+    Returns None for invalid cases.
+    Twitter restricts the length of handles to 15. 16 is the threshold here, since
+    a user might prefix their handle with an '@', a valid case.
+    Tested in tests/test_util.py
+    """
+    if not handle:
+        return None
+
+    parsed_handle = urlparse(handle)
+    if (
+            (parsed_handle.netloc and parsed_handle.netloc != 'twitter.com') or
+            (not parsed_handle.netloc and len(handle) > 16)
+    ):
```

Comment:
Sure. Since this is essentially pattern matching, I felt this style reflects that the best, with the patterns listed out one below the other, un-indented.